### ISSUE TITLE
Requesting 1600kHz shift for Australian 2Mtr Repeaters

### ISSUE THREAD

**Issue by Iantull** [ExpertSDR3/ExpertSDR3-SUGGESTIONS#184]
Australia has both 600kHz and 1600kHz split for their 2Mtr Repeaters but you only allow a maximum of 1000kHz. Could this be increased to 1600kHz.

**Comment by LC1973** [comment]
+1

Plase can this be increased to support +7.6Mhz as is used on 70cm in the UK (through a transverter)

**Comment by LA9AKA** [comment]
Yes, we need the possibility to set more than 1Mhz repeater shift. In LA many 70cm repeaters use 2MHz shift.
